postgres/infoschema.go

```diff
@@ -336,6 +343,83 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 	return primaryKeys, m, nil
 }
 
+type fkConstraint struct {
+	name    string
+	table   string
+	refcols []string
+	cols    []string
+}
+
+// getForeignKeys return list all the foreign keys constraints.
+func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (foreignKeys []schema.ForeignKey, err error) {
+	q := `SELECT
+				schema_name AS "TABLE_SCHEMA",
+ 				cl.relname AS "TABLE_NAME",
+    			att2.attname AS "COLUMN_NAME",
+    			att.attname AS "REF_COLUMN_NAME",
+    			conname AS "CONSTRAINT_NAME"
+			FROM
+   				(SELECT
+        				UNNEST(con1.conkey) AS "parent",
+        				UNNEST(con1.confkey) AS "child",
+        				con1.confrelid,
+        				con1.conrelid,
+        				con1.conname,
+        				ns.nspname AS schema_name
+    				FROM PG_CLASS cl
+        				JOIN PG_NAMESPACE ns ON cl.relnamespace = ns.oid
+        				JOIN PG_CONSTRAINT con1 ON con1.conrelid = cl.oid
+    				WHERE
+						ns.nspname = $1
```

Comment:
Odd indentation on this line and the next?

Reply:
updated indentation